spring-xd-reactor/src/main/java/org/springframework/xd/reactor/ReactorReflectionUtils.java

```diff
@@ -36,22 +38,29 @@
      * be determined.
```

Comment:
2 lines above (can't comment), there is a type. Should read `its`, not `it's`. Also, terminate sentence with a dot.